packages/segmented-control/src/SegmentedControl.tsx

```diff
@@ -279,9 +291,9 @@ const SegmentedControl = React.forwardRef<
   // TODO log warning if defaultValue is set but does not match any child value
   const { usingKeyboard } = useUsingKeyboardContext();
 
-  const [getRef, setRef] = useDynamicRefs<HTMLDivElement>();
+  const getOptionRef = useDynamicRefs<HTMLDivElement>({ prefix: 'option' });
```

Comment:
Curious, did you switch over to this so that you could add the prefix?

Reply:
Not necessarily. Just switched over to the version in `hooks` instead of the older local version